services/scripts/testy-snapshot.sh

```diff
@@ -82,20 +96,24 @@ main() {
 
     # Validate database. Update the snapshot status on success/failure.
     aws ec2 create-tags --resources "$_snapshot_id" "$_volume_id" \
-                        --tags Key=Validation,Value=none
+                        --tags "Key=Validation,Value=none Key=InstanceID,Value=$_instance_id"
 
     echo "Running validation script '$_validation_script' on volume '$_volume_id'."
-    var=$(date +%s%3N)
+    ts=$(date +%s%3N)
     if validate_database "$_validation_script" "$_mount_point" "$_snapshot_id" "$_volume_id"; then
         echo "Successfully validated database backup snapshot '$_snapshot_id'."
         aws logs put-log-events --log-group-name testy-logs \
                                 --log-stream-name testy-logs --log-events \
-            timestamp=$var,message="Backup snapshot validation succeeded. $_snapshot_id. $_instance_id"
+            timestamp=$ts,message="Backup snapshot validation succeeded. $_snapshot_id. $_instance_id"
+        # We can delete the snapshot now it has been validated.
+        if ! aws ec2 delete-snapshot --snapshot-id "$_snapshot_id"; then
```

Comment:
I think it would be good to have a success message - snapshot ____ deleted.

Reply:
Sure, see bb6df2d7d7d972bda9dc51bcfdc38471810c5d02